app/Models/Campaign/Image.php

```diff
@@ -40,9 +41,14 @@ public static function upload($campaign, $img, $name = null)
         return DB::transaction(function() use ($campaign, $img, $name) {
             $path = static::uploadImage($campaign, $img);
             $title = ($name) ? $name : $img->getClientOriginalName();
-
+            $file_name = $img->getClientOriginalName();
```

Comment:
Hi @junaidJaved8461 Cheers for contributing!

In this case i believe the  `$img` should be an instance of `Illuminate\Http\UploadedFile`  so you shouldn't need to grab/move the file at all here. In theory type & size should be grabbable via the `->getMimeType()` and `getSize()`.

In terms of storage, it'd be best to store without the `kb` if possible also, that way we can easily aggregate the size of all the uploaded files on a campaign for example, by calling sum on the sizes. (The kb part can then be added in the UI, or accessor if we always want it).